tests/gcp/unit_tests/google_service_account/main.tf

```diff
@@ -0,0 +1,19 @@
+data external test_disabled {
```

Comment:
Can we test getting an IAM conditions field instead. That's a more complex algorithm.